docs/integrations/databases/MotherDuck.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "MotherDuck"
+sidebarTitle: "MotherDuck"
+description: "Execute SQL commands against a MotherDuck database."
+---
+<div align="center">
+<img src="https://github.com/mage-ai/mage-ai/assets/59450879/0e410c12-73a0-43d8-97fe-396772d2767b" width="60%"/>
+</div>
+
```

Comment:
Let's mention this feature is supported in version >= 0.9.63

Reply:
Sounds good